tests/mail/tests.py

```diff
@@ -2441,7 +2832,7 @@ def test_encodes_idna_in_smtp_commands(self):
                 "To": "Discussão Django <django@discussão.example.org>",
             },
         )
-        backend = smtp.EmailBackend()
+        backend = smtp.EmailBackend(**settings.EMAIL_PROVIDERS["default"]["OPTIONS"])
```

Comment:
```suggestion
        backend = mail.get_connection()
```

(And similar in other tests.)